packages/webpack/package.json

```diff
@@ -22,7 +22,7 @@
     "eventsource-polyfill": "^0.9.6",
     "extract-css-chunks-webpack-plugin": "^4.5.2",
     "file-loader": "^4.0.0",
-    "fs-extra": "^8.0.1",
+    "fs-extra": "^7.0.1",
```

Comment:
It seemed that webpack no longer depends on fs-extra,  this can be removed

Reply:
Yeah